scripts/create-kind-cluster-with-SSL-passthrough.sh

```diff
@@ -146,25 +149,22 @@ else
   kubectl --context "kind-${name}" apply -f "$NGINX_INGRESS_URL"
 fi
 
+echo "Waiting for nginx ingress deployment to be created..."
+kubectl --context "kind-${name}" wait --for=create deployment/ingress-nginx-controller \
+  --namespace ingress-nginx --timeout=300s
```

Comment:
If I understand correctly, this wait is unnecessary because the preceding `kubectl apply` command will not return until after the `Deployment` object has been created.